lib/src/quality/apps/text/links.py

```diff
@@ -0,0 +1,43 @@
+import pandas as pd
+import requests
+from malevich.square import DF, Context, processor, scheme
+from pydantic import BaseModel
+
+
+@scheme()
+class Links(BaseModel):
+    link: str
+
+@processor()
+def asser_links(df: DF[Links], context: Context):
+    """
+        Check if the links are valid (do not return error status codes)
+
+        Input:
+          DF[Links] - DataFrame with one column named 'link'.
+
+        Output:
+          DF[Links] - DataFrame with one column named 'link' containing
+          valid or invalid links depending on 'filter_links' option.
+
+        Options:
+          filter_links - If True, returns links which passed assert. Else return
+          error links.
+    """
```

Comment:
Do not forget to include

```
...

--- <-- put this three lines to make horizontal separator both in IDE and UI.

Args:
    ...

Returns:
    ...
    
```

See https://google.github.io/styleguide/pyguide.html